_data/oh1addons.csv

```diff
@@ -57,6 +57,7 @@ binding,Nikobus Binding
 binding,Novelan Heatpump Binding
 binding,OpenEnergyMonitor Binding
 binding,OneWire Binding
+binding,Owntracks (mqttitude) Binding
```

Comment:
you can leave it, it will be automatically overwritten once I update the generated content.